### ISSUE TITLE
Unable to start a REPL inside a project.

### ISSUE THREAD

**Issue by eynhcaysem** [technomancy/leiningen#2774]
**Initial debugging steps**
Before creating a report, _especially_ around exceptions being thrown when running Leiningen, please check if the error still occurs after:

- [x] Updating to using the latest released version of Leiningen (`lein upgrade`).
- [x] Moving your `~/.lein/profiles.clj` (if present) out of the way. This contains third-party dependencies and plugins that can cause problems inside Leiningen.
- [x] Updating any old versions of plugins in your `project.clj`, especially if the problem is with a plugin not working. Old versions of plugins like nREPL and CIDER (as well as others) can cause problems with newer versions of Leiningen.
- [x] (If you are using Java 9 or newer), updating your dependencies to their most recent versions. Recent JDK's have introduced changes which can break some Clojure libraries.

**Describe the bug**
Lein repl doesn't start when inside a certain project. This project doesn't have any dependencies in project.clj other than the latest version of clojure (1.10.3). Without a project repl starts with no issues. Creating a new project and starting a repl there also works. This issue appeared seemingly out of nowhere: previously repl worked in this project.

**To Reproduce**
1. Go to a certain lein project
2. Run the command lein repl
3. See error

**Actual behavior**
An exception is thrown and the process exits:

$ lein run
Syntax error compiling at (/tmp/form-init11477894767259776810.clj:4341:33).
Unable to resolve symbol: create in this context

Full report at:
/tmp/clojure-4019537888258490872.edn
Subprocess failed (exit code: 1)

**Expected behavior**
A lein repl server should start.

**Logs**
https://gist.github.com/eynhcaysem/7be82bb5479e1d34a96776b2e16a0c7b

**Environment**
- Leiningen Version: Leiningen 2.9.7 on Java 11.0.11 OpenJDK 64-Bit Server VM
- Leiningen installation method: manual.
- JDK Version: openjdk version "11.0.11" 2021-04-20
- OS: Ubuntu 20.04.3 LTS

**Comment by technomancy** [comment]
> Steps to reproduce the behavior:
>
> Go to a certain lein project
> Run the command lein repl
> See error

I think you might be a little confused about what "steps to reproduce" means.

In order for a bug report to be useful, it needs to contain enough information for other people to be able to see the bug.

**Comment by eynhcaysem** [comment]
Ah, yes, sorry, this is my first time opening an issue.

And it also seems like I opened it for no reason. I've tried to actually create a reproducible example and found out that somehow one letter was changed in the ns declaration in core.clj, which took me like two hours to spot. The stack trace didn't help as it didn't mention any of my code, so I initially assumed that it was a problem with Leiningen. My bad.
